### PR TITLE
fix(wallet-mobile): uat updates

### PR DESCRIPTION
**Pull request by banklesss**
##### Tickets

YOMO-1892
YOMO-1897
YOMO-1903
YOMO-1905
YOMO-1906
YOMO-1907
YOMO-1908

### CODE REVIEW

File: apps/wallet-mobile/src/components/Button/Button.tsx [hunk 1]
```diff
@@ -186,6 +191,12 @@ const useStyles = () => {
       ...atoms.p_sm,
       ...atoms.body_2_md_medium,
     },
+    warningTheme: {
+      backgroundColor: color.sys_magenta_500,
+    },
+    warningThemeText: {
+      color: color.gray_min,
```
Comment (michalszorad) [inline apps/wallet-mobile/src/components/Button/Button.tsx:198]: Should this be `text_...`?

File: apps/wallet-mobile/src/legacy/Staking/PoolTransition/PoolTransitionModal.tsx [hunk 2]
```diff
@@ -39,101 +39,99 @@ export const PoolTransitionModal = ({
   }
 
   const timeSpan = poolTransition.deadlineMilliseconds - Date.now()
-  const isActive = timeSpan > 0
+  const isActive = false
```
Comment (jorbuedo) [inline apps/wallet-mobile/src/legacy/Staking/PoolTransition/PoolTransitionModal.tsx]: Was this for testing?

File: apps/wallet-mobile/src/features/Portfolio/useCases/PortfolioScreen.tsx [hunk 3]
```diff
@@ -80,7 +80,7 @@ export const PortfolioScreen = () => {
                     ListHeaderComponent={() => {
                       return (
                         <View>
-                          <Text>walelt: {walletId}</Text>
+                          <Text>wallet: {walletId}</Text>
```
Comment (michalszorad) [inline apps/wallet-mobile/src/features/Portfolio/useCases/PortfolioScreen.tsx:83]: Should this use translation?

File: apps/wallet-mobile/src/legacy/Staking/PoolTransition/PoolTransitionModal.tsx (hunk 2 above)
Comment (banklesss) [reply apps/wallet-mobile/src/legacy/Staking/PoolTransition/PoolTransitionModal.tsx]: yeah !!!! 👀

File: apps/wallet-mobile/src/features/Portfolio/useCases/PortfolioScreen.tsx (hunk 3 above)
Comment (banklesss) [reply apps/wallet-mobile/src/features/Portfolio/useCases/PortfolioScreen.tsx:83]: it is a playground

File: apps/wallet-mobile/src/legacy/Staking/PoolTransition/PoolTransitionModal.tsx (hunk 2 above)
Comment (banklesss) [inline apps/wallet-mobile/src/legacy/Staking/PoolTransition/PoolTransitionModal.tsx]:
```suggestion
  const isActive = timeSpan > 0
```

File: apps/wallet-mobile/src/legacy/Staking/PoolTransition/PoolTransitionModal.tsx [hunk 4]
```diff
@@ -224,38 +226,39 @@ const useStyles = () => {
...
     },
     line: {
-      width: '120%',
       height: StyleSheet.hairlineWidth,
       backgroundColor: color.gray_200,
     },
     actions: {
-      alignSelf: 'stretch',
+      ...atoms.px_lg,
+      ...atoms.align_stretch,
```
Comment (banklesss) [inline apps/wallet-mobile/src/legacy/Staking/PoolTransition/PoolTransitionModal.tsx]:
```suggestion
      alignSelf: 'stretch',
```

**Review by jorbuedo — APPROVED** [review]

File: apps/wallet-mobile/src/components/Button/Button.tsx (hunk 1 above)
Comment (banklesss) [reply apps/wallet-mobile/src/components/Button/Button.tsx:198]: not defined : ( https://www.figma.com/design/Ey4FrVVzQVBb8tXJCnfvYC/Color?node-id=1672-221&node-type=FRAME&m=dev

**Pull request merged**